roles/satellite-clone/files/disassociate_capsules.rb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/ruby
+username = "admin"
+password = "changeme"
```

Comment:
@bbuckingham We reset the password to admin/changeme [here](https://git.io/vS4xlZ) so it will be these values at this point in the playbook